SCSH_3C_iOS/SCSH_3C_iOS/Modules/Home/View/HomeViewController.swift

```diff
@@ -107,6 +85,46 @@ class HomeViewController: UIViewController {
             bodyCollectionView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
         ])
     }
+    
+    private func bindEvents() {
+        viewModel.themes
+            .zip(viewModel.firstHeaderIndex)
+            .throttle(for: .milliseconds(100), scheduler: DispatchQueue.main, latest: true)
+            .receive(on: DispatchQueue.main)
+            .sink { [weak self] _, firstIndex in
+                self?.headerCollectionView.reloadData()
+                self?.bodyCollectionView.reloadData()
```

Comment:
I'm not sure, but it seems like a sequence of `reloadData()`.

What side effects or conflicts might arise if we use *Async* for handling their respective data?

Reply:
**Based on the conditions below I wrote the reloadData() in the same block.**

1. `headerCollectionView` and `bodyCollectionView` share the same data source.
2.  neither can be updated without the other, or it is ineffective.

>  if we use Async for handling their respective data?
I can't understand this one, could you describe more detail?

Reply:
If there's a dependency, that's fine.

I assumed these could be two separate standalone tasks, at first. In that case, can run them asynchronously.